python/ray/train/tests/test_gpu.py

```diff
@@ -19,6 +20,40 @@ def ray_start_4_cpus_2_gpus():
     ray.shutdown()
 
 
+def test_torch_auto_gpu_to_cpu(ray_start_4_cpus_2_gpus):
+    """Tests if GPU tensors are auto converted to CPU on driver."""
+
+    num_workers = 2
+
+    def train_func():
+        model = torch.nn.Linear(1, 1)
+
+        # Move to GPU device.
+        model = ray.train.torch.prepare_model(model)
+
+        assert next(model.parameters()).is_cuda
+
+        ray.train.save_checkpoint(model=model)
+
+    trainer = Trainer("torch", num_workers=num_workers, use_gpu=True)
+    trainer.run(train_func)
+    trainer.shutdown()
```

Comment:
Do you need to check that `trainer.latest_checkpoint` is on CPU here?

Reply:
Done!